components/organisms/light-cards-image.tsx

```diff
@@ -0,0 +1,20 @@
+import { lightCardsImage } from '../../data/light-cards-image'
+import CardImage from '../molecules/card-image'
+
+export default function lightServices () {
+  return (
+    <section className='section-padding bg-blue-lighter'>
```

Comment:
Got my answer... You can merge lightServices -> and DarkCardsImage into one component.

Please make sure that you component file name and function name match